configure.ac

```diff
@@ -393,6 +393,14 @@ AC_ARG_ENABLE([arm64-crc32], AS_HELP_STRING([--disable-arm64-crc32],
 		is detected.]),
 	[], [enable_arm64_crc32=yes])
 
```

Comment:
An extra newline is needed here to match the code style. It helps break up the different sections in this file.